Doc/library/datetime.rst

```diff
@@ -1550,7 +1550,7 @@ Instance methods:
 Examples of Usage: :class:`.datetime`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Examples of working with :class:`~datetime.datetime` objects:
+Examples of working with :class:`~.datetime.datetime` objects:
```

Comment:
Does this work?
```suggestion
Examples of working with :class:`.datetime` objects:
```
Or `~` is needed?
```suggestion
Examples of working with :class:`~.datetime` objects:
```

Reply:
I'm sorry. I'm still trying to assimilate all the Sphinx line noise. I corrected this.